packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.base.tsx

```diff
@@ -1,10 +1,11 @@
 import * as React from 'react';
+
+import { ITextField, TextField } from '../../TextField';
 import { BaseComponent, classNamesFunction } from '../../Utilities';
+import { getColorFromRGBA, getColorFromString, IColor, MAX_COLOR_HUE, updateA, updateH, updateSV } from '../../utilities/color/colors';
```

Comment:
These changes to imports are pretty simple, but they're pretty hard to parse as humans since the changes aren't needed (there's no changes, right?). Maybe there's a prettier / plugin that's doing these re-orders for you?

Reply:
Thanks for catching this, it's caused by the TypeScript Hero extension I have running. This is fine for new files but there's no reason to introduce unrelated changes in PRs. I'll revert these. 😄